lib/engine/game/g_18_fl/step/token.rb

```diff
@@ -37,7 +37,8 @@ def process_place_token(action)
 
             unless @game.loading
               verified_cost = token_cost_override(action.entity, action.city, action.slot, action.token)
-              raise GameError, 'Error verifying token cost; is game out of sync?' unless action.cost == verified_cost
+              raise GameError, "Error verifying token cost; is game out of sync? ID:#{action.id}" unless
```

Comment:
can you undo the debug change to the exception

Reply:
I've undone it